include/FileBrowser.h

```diff
@@ -134,8 +137,12 @@ class FileBrowserTreeWidget : public QTreeWidget
 	QPoint m_pressPos;
 
 	//! This should only be accessed or modified when m_pphMutex is held
+	#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))
+		QRecursiveMutex m_pphMutex;
+	#else
+		QMutex m_pphMutex;
+	#endif
```

Comment:
```suggestion
#if (QT_VERSION >= QT_VERSION_CHECK(5,15,0))
	QRecursiveMutex m_pphMutex;
#else
	QMutex m_pphMutex;
#endif
```
Also you moved this a live above, meaning the doc comment is no longer for `m_previewPlayHandle` as it should be. Move this back below please.